lib/ansible/modules/windows/win_get_url.py

```diff
@@ -57,49 +55,53 @@
         resource, so for this to work, the remote web server must support
         HEAD requests.
     version_added: "2.0"
-    required: false
-    choices: [ "yes", "no" ]
-    default: yes
+    type: bool
+    default: 'yes'
   username:
     description:
       - Basic authentication username
-    required: false
-    default: null
   password:
     description:
       - Basic authentication password
-    required: false
-    default: null
   skip_certificate_validation:
     description:
     - This option is deprecated since v2.4, please use C(validate_certs) instead.
     - If C(yes), SSL certificates will not be validated. This should only be used
       on personally controlled sites using self-signed certificates.
-    default: 'no'
     type: bool
+    default: 'no'
   validate_certs:
     description:
     - If C(no), SSL certificates will not be validated. This should only be used
       on personally controlled sites using self-signed certificates.
     - If C(skip_certificate_validation) was set, it overrides this option.
```

Comment:
Wouldn't we want to opposite where `validate_certs` is the source of truth considering `skip_certificate_validation` will be deprecated?